dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -147,74 +145,27 @@ public DoraLoadJob(
     mBandwidth = bandwidth;
     mUsePartialListing = usePartialListing;
     mVerificationEnabled = verificationEnabled;
-    FileSystem fs = FileSystem.Factory.create(Scheduler.getInstance().getFileSystemContext());
-    ListStatusPOptions listOptions = ListStatusPOptions
-        .newBuilder()
-        .setRecursive(true).build();
-    mFs = fs;
-    mFileIterator = Optional.of(new FileListFetcher(fs, mPath, listOptions, false));
-  }
+    String ufsRoot = Configuration.getString(PropertyKey.DORA_CLIENT_UFS_ROOT);
+    AlluxioURI ufsRootUri = new AlluxioURI(ufsRoot);
+    mUfs = UnderFileSystem.Factory.create(
```

Comment:
@lucyge2022 please review this approach

Reply:
think this is better than ufsmanager as it doesnt keep in cache like ufsmanager does @jja725 
but make sure u close the mUfs on exception / job exiting

Reply:
honestly i don't know how to capture the job exiting event. Should i close the connection in failJob() and setJobSuccess() methods?